cfgov/jinja2/v1/events/_macros.html

```diff
@@ -93,7 +93,7 @@
         {% endmacro %}
 
         {% macro _zip() %}
-            <span class="event-meta_zip p-postal-code">{{ zip }}</span>
+            <span class="event-meta_zip p-postal-code">{{ '%05d' | format(zip) }}</span>
```

Comment:
It might be an issue if someone accidentally put in 4 digits instead of 5 because we aren' t currently aren't validating the zip.